Unity-Technologies/com.unity.multiplayer.samples.bitesize
# [MTT-9094] [MTT-8906] [MTT-9092] Integrate ingame UI

### Description

This PR adds ingame UI. 

#### Character UI:

- Will display the PlayerName (currently shows Gameobject Name) will be changed in the follow up.
- Displays small hint when the user carries around objects.

<img width="417" alt="Screenshot 2024-10-15 at 15 34 11" src="https://github.com/user-attachments/assets/ba262710-0f22-4d1e-a3a8-93b0b2ad01ad">

#### Pickup UI:
- Displayed when the Character is in range of a pickup element
<img width="282" alt="Screenshot 2024-10-15 at 15 38 24" src="https://github.com/user-attachments/assets/40624b2d-2620-47ab-a33a-2f1967d89c51">

#### Ingame Menu (accessible via Button on left top):
<img width="388" alt="Screenshot 2024-10-15 at 15 34 00" src="https://github.com/user-attachments/assets/fdc82afa-fb1f-4059-8acb-1ebd491eca3e">

#### Main Screen has been aligned with new Design

new:
<img width="1438" alt="Screenshot 2024-10-15 at 15 33 16" src="https://github.com/user-attachments/assets/afa69f2e-e15f-4552-a385-6f00ab288069">

old:
<img width="1487" alt="Screenshot 2024-10-01 at 17 12 53" src="https://github.com/user-attachments/assets/024ce031-8d59-4ed4-82dc-a8b8700115d9">




How to test:
Open Bootstrap scene and press play (after connecting Vivox in project settings to a valid project id).
Check that entering Player name, Session name and pressing Start button in Main Menu work as expected.
Menus can be navigated with TAB aswell as with Mouse. Play around by picking up items etc. should be tested with multiple players idealy

### Issue Number(s)
https://jira.unity3d.com/browse/MTT-9094
https://jira.unity3d.com/browse/MTT-8906
https://jira.unity3d.com/browse/MTT-9092
